diffs/AaveV3Ethereum_EnableSUSDeUSDTLiquid_20241125_before_AaveV3Ethereum_EnableSUSDeUSDTLiquid_20241125_after.md

```diff
@@ -0,0 +1,36 @@
+## Emodes changed
+
+### EMode: ETH correlated(id: 1)
```

Comment:
can you rebase on latest main and run `npm i` & rerun the tests? Just curios as these unchanged eModes should disappear from the report.